new-packaging/README.md

```diff
@@ -0,0 +1,21 @@
+You need `xmlstarlet`, `dos2unix`
+
+From `/path/to/neo4j` root directory run `mvn install`
+
+`mvn clean install -DskipTests`
+
+you need `cypher-shell` packaged
+
+`make out/cypher-shell.zip`
+
+and then,
+
+`cp out/cypher-shell.zip /path/to/neo4j`
+
+From `/path/to/neo4j/packaging directory` run `mvn install`
+
+`mvn clean install -DskipTests`
+
+From `/path/to/neo4j/new-packaging`
+
+`PATH=$(pwd)/bin:/Applications/install4j.app/Contents/Resources/app/bin/:$PATH DISTRIBUTION=stable make clean neo4j-desktop-windows`
```

Comment:
That path is mac specific with regards to the install4j.

It also only refers to the desktop installer.

Maybe divide this into different sections

```
## Debian

PATH=$(pwd)/bin:$PATH DISTRIBUTION=stable make clean debian

## RPM

PATH=$(pwd)/bin:$PATH DISTRIBUTION=stable make clean rpm

## Windows Deskop installer

PATH=$(pwd)/bin:/path/to/install4j/bin:$PATH DISTRIBUTION=stable make clean neo4j-desktop-windows
```